src/pages/tutorials/library.tsx

```diff
@@ -0,0 +1,113 @@
+import algoliasearch from 'algoliasearch/lite'
+import { InstantSearch } from 'react-instantsearch-hooks-web'
+import { history } from 'instantsearch.js/es/lib/routers'
+import { useRouter } from 'next/router'
+
+import SidebarSidecarLayout, {
+	SidebarSidecarLayoutProps,
+} from 'layouts/sidebar-sidecar'
+import TutorialsLibraryView from 'views/tutorial-library'
+import { ConnectedTutorialLibraryFilters } from 'views/tutorial-library/components/filters'
+import { INDEX_NAME } from 'views/tutorial-library/constants'
+import {
+	routerStateToSearchState,
+	searchStateToRouteState,
+} from 'views/tutorial-library/utils/router-state'
+import SidebarNavList from 'components/sidebar/components/sidebar-nav-list'
+import { SidebarNavMenuItem } from 'components/sidebar/components'
+import { generateTopLevelSubNavItems } from 'lib/generate-top-level-sub-nav-items'
+import { getTutorials } from 'lib/learn-client/api/tutorial'
+import { Tutorial } from 'lib/learn-client/types'
+import { stripUndefinedProperties } from 'lib/strip-undefined-props'
+
+const DEFAULT_SLUGS = [
+	'terraform/infrastructure-as-code',
+	'vault/getting-started-intro',
+	'consul/get-started',
+	'nomad/get-started-install',
+	'vagrant/getting-started-index',
+	'packer/get-started-install-cli',
+	'boundary/getting-started-intro',
+	'waypoint/get-started-intro',
+	'cloud/terraform-hcp-provider-vault',
```

Comment:
💭 should these be authorable?

Reply:
could be a future thing.

Reply:
Yeah that could be cool, just going 1:1 with Learn as much as possible given it's September 20th ;)

Reply:
Fair enough!

Reply:
Yeah, I think eventually we can/will pull them out. For now, it seems like we can align with what learn has now just to keep the scope limited.